astro.config.mjs

```diff
@@ -508,6 +508,10 @@ export default defineConfig({
               collapsed: true,
               autogenerate: { directory: '/snowflake/integrations' },
             },
+            {
```

Comment:
do we want to maintain the same page order in the navigation panel as in AWS docs?